crates/bevy_render/src/camera/camera.rs

```diff
@@ -510,8 +510,8 @@ impl Default for CameraOutputMode {
 }
 
 /// Configures the [`RenderGraph`](crate::render_graph::RenderGraph) name assigned to be run for a given [`Camera`] entity.
-#[derive(Component, Deref, DerefMut, Reflect, Clone)]
-#[reflect_value(Component)]
+#[derive(Component, Debug, Deref, DerefMut, Reflect, Clone)]
+#[reflect_value(Component, Debug)]
 pub struct CameraRenderGraph(InternedRenderSubGraph);
```

Comment:
This only holds a `Interned<dyn RenderSubGraph>` and that doesn't have Debug, so I'm not sure how useful this actually is? Is it just so it can print the type name?

Reply:
That sounds like @SludgePhD's stated goal of "the name of the configured render graph should be visible in editors, etc." would not be reached with this PR 🤔

Reply:
The type name of the specific render graph label type that was used, yes. Without this change the debug representation just says something like `"CameraRenderGraph"`. At least I *think* that's how interned labels work?

Reply:
Oh, okay, just tested it and Core3d prints `CameraRenderGraph(Core3d)`. Nice, I didn't realize the interned stuff worked like that.